extensions-contrib/distinctcount/src/main/java/io/druid/query/aggregation/distinctcount/DistinctCountAggregator.java

```diff
@@ -22,13 +22,15 @@
 import io.druid.collections.bitmap.MutableBitmap;
 import io.druid.query.aggregation.Aggregator;
 import io.druid.segment.DimensionSelector;
+import io.druid.segment.NullHandlingHelper;
 import io.druid.segment.data.IndexedInts;
 
 public class DistinctCountAggregator implements Aggregator
 {
-
+  private static final int UNKNOWN_ID = -1;
```

Comment:
This value shouldn't exist, I suggest just to not set `idForNull` if `nameLookupPossibleInAdvance()` is false, because `idForNull` is unused in this case